src/ui/Combobox/Combobox.ts

```diff
@@ -108,7 +107,17 @@ export class Combobox implements ICombobox {
     this.input.updateAccessibilityAttributes(attributes);
   }
 
-  public updateAriaLive(text: string) {
+  public updateAriaLive() {
+    if (!this.values.hasValues()) {
+      this.options.ariaLive.updateText(l('NoValuesFound'));
+      return;
+    }
+
+    let text = l('ShowingResultsWithQuery', this.values.numberOfValues, this.input.value, this.values.numberOfValues);
+    if (this.options.scrollable && this.options.scrollable.areMoreValuesAvailable()) {
+      text = `${text}. ${l('MoreValuesAvailable')}`;
```

Comment:
is putting a `.` wrong? 🤔

Reply:
@fbeaudoincoveo

Reply:
I'm not against using a `.`

If we'd rather avoid it, another option would be to display the `MoreValuesAvailable` string between parentheses instead, e.g.,

> 5 results for my query (additional values are available)
> 5 résultats pour ma requête (des valeurs additionnelles sont disponibles)

This implies that `MoreValuesAvailable` would have to begin with a lowercase letter (and preferably shouldn't end with a period). I'd personally go for the `.`, though :)

Another thing I noticed is that at the moment, we don't seem to be emphasizing the input value in any way. Perhaps we could surround it with single quotes? I think this would read slightly better:

> 5 results for 'my query'. Additional values are available.
> 5 résultats pour 'ma requête'. Des valeurs additionnelles sont disponibles.